conda_build/skeletons/cran.py

```diff
@@ -520,8 +520,18 @@ def get_latest_git_tag(config):
     return tags[-1]
 
 
+def _ssl_no_verify():
+    """Gets whether the SSL_NO_VERIFY environment variable is set to 1 or True.
+
+    This provides a workaround for users in some corporate environments where
+    MITM style proxies make it difficult to fetch data over HTTPS.
+    """
+    return environ.get('SSL_NO_VERIFY', '').strip().lower() in ('1', 'true')
+
+
 def get_session(output_dir, verbose=True):
     session = requests.Session()
+    session.verify = _ssl_no_verify()
```

Comment:
I've just noticed that I made an error here, there should be a `not` before the function call. Apologies. How should I go about rectifying this, another pull request?

Reply:
Yeah, just another PR.  If you can, please add a test to make sure that bugs like this logic error don't creep in.